openapi/azure.openai.embeddings/Module.md

```diff
@@ -0,0 +1,67 @@
+## Overview
+This is a generated connector from [Azure OpenAI Embeddings API v2023-03-15-preview](https://learn.microsoft.com/en-us/azure/cognitive-services/openai/reference#embeddings/) OpenAPI specification.
+
+The Azure Azure OpenAI Service REST API Embeddings Endpoint will generate one a vector representation of a given input that can be easily consumed by machine learning models and other algorithms.
+
+## Prerequisites
+- Create an [Azure](https://azure.microsoft.com/en-us/features/azure-portal/) account
+- Create an [Azure OpenAI resource with a model deployed](https://learn.microsoft.com/en-us/azure/cognitive-services/openai/how-to/create-resource)
+- Obtain tokens
+    - Use [this](https://learn.microsoft.com/en-us/azure/cognitive-services/openai/reference#authentication) guide to learn how to generate and use tokens
+
+## Quickstart
+
+To use the Azure OpenAI Embeddings connector in your Ballerina application, update the .bal file as follows:
+
+### Step 1: Import connector
+Import the `ballerinax/azure.openai.embeddings` module into the Ballerina project.
+
+```ballerina
+import ballerinax/azure.openai.embeddings;
+```
+
+### Step 2: Create a new connector instance
+
+Create and initialize an `embeddings:Client` with the obtained `apiKey` and a `serviceUrl` containing the deployed models.
+
+    ```ballerina
+
+    final embeddings:Client embeddingsClient = check new (
+        config = {auth: {apiKey: apiKey}},
+        serviceUrl = serviceUrl
+    );
+
+    ```
+
+### Step 3: Invoke connector operation
+1. Now you can use the operations available within the connector. Note that they are in the form of remote operations.
+
+    Following is an example on obtaining embeddings from a GPT-3 ada model::
+
+    Obtain embeddings from ada model
```

Comment:
I wrote it to be similar to the OpenAI Embeddings connector. Shall we have it and not mention a specific model?

Reply:
Since we have mentioned `Following is an example on obtaining embeddings from a GPT-3 ada model` above I don't think we need to say `Obtain embeddings from ada model` and `Sample` again. Insert comments to the code if needed.

Reply:
Ack. Will do

Reply:
Resolved through a185539b01a4cf4742a0e93829edb24003bdbef7